src/array.rs

```diff
@@ -86,6 +89,11 @@ pub trait JavaArrayExt<T: JavaType>: JvmOp {
     fn length(self) -> Self::Length;
 }
 
+pub trait JavaArrayModificationExt<T: JavaType, RT: JavaScalar>: JvmOp {
+    type SetArrayRegion<'a>: VoidMethod;
+    fn set_array_region<'a>(self, start: usize, values: &'a[RT]) -> Self::SetArrayRegion<'a>;
```

Comment:
we need to clearly document the invariants that the caller needs to uphold. Is this function always safe to call even if the values are too long, start is out of bounds, etc.? We need tests of that as well